src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs

```diff
@@ -279,5 +279,26 @@ public static void Equals_Compare_Success()
             Assert.True(ip4.GetHashCode().Equals(ip5.GetHashCode()));
             Assert.False(ip4.GetHashCode().Equals(ip6.GetHashCode()));
         }
+
+#if NetStandard17
```

Comment:
> remember david mentioning that new test files should not be added , but conditional include the test in the existing file as i did

I made that comment before I was told that adding new files the "_netstandard1.7" suffix was the agreed-upon standard.  I was also told that it was a temporary thing and that soon these types of differences will be removed as the [master] branch moves toward only building for netstandard1.7.